src/org/ods/services/OpenShiftService.groovy

```diff
@@ -431,20 +433,16 @@ class OpenShiftService {
         ).trim()
     }
 
-    boolean checkForExistingValidDeploymentBasedOnStoredConfig (Map repo, boolean forceOverride = false) {
+    boolean isOCPDeploymentBasedOnLatestRepoState (Map repo) {
```

Comment:
This might answer my question about `amendRepoForResurrection` ... it seems this method actually modifies `repo.data.odsBuildArtifacts`. I would prefer this method not to do that, or that we rename the method to make things clear. Right now I find it surprising :)

Reply:
check -will do :)